tox.ini

```
@@ -59,17 +59,56 @@ commands =
 deps =
     -r{toxinidir}/requirements/quality.txt
 commands =
-    black --check --diff --line-length=80 src/scriv tests docs setup.py
+    black --check --diff --line-length=80 src/scriv tests docs
     python -m cogapp -cP --check --verbosity=1 docs/*.rst
     mypy src/scriv tests
-    pylint src/scriv tests docs setup.py
-    pycodestyle src/scriv tests docs setup.py
-    pydocstyle src/scriv tests docs setup.py
-    isort --check-only --diff -p scriv tests src/scriv setup.py
-    python setup.py -q sdist
+    pylint src/scriv tests docs
+    pycodestyle src/scriv tests docs
+    pydocstyle src/scriv tests docs
+    isort --check-only --diff -p scriv tests src/scriv
+    python -m build --sdist
```

Why not lint the wheel too?
```suggestion
    python -m build
```

---

Why did you add -I?

---

Just a habit. I add it to all `python` invocations. `-m` auto-adds CWD to `$PYTHONPATH` and this may guard against implicitly importing something that you didn't mean. It's especially useful with `python -Im pytest` but it might be handy in various other contexts, so I just default to it.